app/llm_integration.py

```
@@ -0,0 +1,740 @@
+                summary = summary.split("```json", 1)[1]
+            if summary.startswith("```"):
+                summary = summary.split("```", 1)[1]
+            if summary.endswith("```"):
+                summary = summary.rsplit("```", 1)[0]
+            
+            summary = summary.strip()
+            
+            # Validate JSON
+            json.loads(summary)
+            return summary
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)